client/src/components/App/Carousel.test.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { fireEvent, render, screen } from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
```

Comment:
nit: unused. This doesn't really matter, but might want to put in a lint rule to prevent unused imports

Reply:
My linter is already screaming at me over so many things :(. I do have an unused imports rule, might've just ignored it. Which import can I remove? The one on line 4?